### PR TITLE
DAT-15191. added valueSequenceNext test case for Snowflake

### PR DESCRIPTION
**Pull request by PavloTytarchuk**

### CODE REVIEW

**Review by KushnirykOleh — APPROVED** [review]
good to merge after corresponding PR in core is merged.

**Comment by filipelautert** [comment]
Core corresponding branch was merged.

**Comment by KushnirykOleh** [comment]
green build https://github.com/liquibase/liquibase-test-harness/actions/runs/6081854858

**Pull request merged**
